services-directions/src/main/java/com/mapbox/api/directions/v5/MapboxDirections.java

```diff
@@ -1046,6 +1046,22 @@ public Builder waypointTargets(@NonNull List<Point> waypointTargets) {
 
     abstract Builder waypointTargets(@Nullable String waypointTargets);
 
+    /**
+     * A point to specify drop-off locations that are distinct from the locations specified in
+     * coordinates.
+     * The number of waypoint targets must be the same as the number of coordinates,
+     * but you can skip a coordinate with a null value.
+     * Must be used with steps=true.
+     *
+     * @param waypointTarget a point for drop-off locations
+     * @return this builder for chaining options together
+     * @since 4.3.0
```

Comment:
```suggestion
     * @since 5.9.0
```

Reply:
Wait, addWaypointTarget is not being added, github made this look weird. addSnappingClosure is what is added

Reply:
this is not related to an added method, I just moved `addWaypointTarget `

Reply:
Next time you can remove `@since` altogether. It's a maintenance nightmare 😅